### PR TITLE
Sittercity QA Blog Post - 20150210

### PR DESCRIPTION
**Pull request by ravenduo**

### CODE REVIEW

File: _posts/2015-02-10-sittercity-quality-assurance.markdown [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  - Over 1000 automated test cases.
+  - Over 10000 automated testing steps.
+  - In 2014 over 100,000 test cases were executed.
+
+### How do we accomplish this?
+We are able to accomplish all of this with a variety of tools and processes.
+
+![Sittercity QA](/assets/qa_tools.jpg)
+
+ - [Cucucmber](https://cukes.info/) has been an excellent acceptance criteria framework used across our product and development teams to describe desired functionality.  These are easily translated into unit and full stack acceptance tests that the development and QA team can build and maintain.
```
Comment (ppeble) [inline _posts/2015-02-10-sittercity-quality-assurance.markdown]: Typo - 'Cucucmber'

File: _posts/2015-02-10-sittercity-quality-assurance.markdown [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ - [Calabash](http://calaba.sh/) is used for our automated mobile testing.  It can be used across iOS and Android platforms.  It is tightly integrated with Cucumber.
+ - [WatirWebdriver](http://watirwebdriver.com/) is the engine behind our full stack Cucucmber tests.  WatirWebdriver allows our tests to run in actual browsers, locally or using cloud based virtual machines.
```
Comment (ppeble) [inline _posts/2015-02-10-sittercity-quality-assurance.markdown]: Another typo - 'Cucucmber'

File: _posts/2015-02-10-sittercity-quality-assurance.markdown [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Sittercity Quality Assurance"
+date:   2015-02-10 03:45:22
+categories: jekyll update
```
Comment (ppeble) [inline _posts/2015-02-10-sittercity-quality-assurance.markdown]: I think you can take 'jekyll' and 'update' out of this. This affects the actual link that is generated (it actually creates `http://<url>/jekyll/update/2015/02/10/sittercity-quality-assurance.html`). You can probably just put 'qa'.

**Comment by jazminschroeder** [comment]
Looks great. I like the typical qa day image a lot!

Some ideas: Add a section at the end to invite readers to leave comments, share tooling, etc. Maybe invite readers to future posts where we may dive into a particular tool?

**Comment by ppeble** [comment]
FYI, we are not allowing comments anymore.

**Comment by ppeble** [comment]
:sheep:

**Pull request merged**